cli/src/experimental/tasks/update_stage.rs

```diff
@@ -135,26 +107,15 @@ fn fetch_workflow(
     Ok(workflow)
 }
 
-fn update_task_stage(
-    client: &Client,
-    host: &str,
-    token: &crate::utils::auth::Token,
-    task_id: &Uuid,
-    stage_id: &Uuid,
-) -> Result<()> {
-    let url = format!(
-        "{}/api/environments/{}/tasks/{}/update_stage/",
-        host, token.env_id, task_id
-    );
+fn update_task_stage(client: &PHClient, task_id: &Uuid, stage_id: &Uuid) -> Result<()> {
+    let url = format!("tasks/{}/update_stage/", task_id);
 
     let request_body = UpdateStageRequest {
         current_stage: *stage_id,
     };
 
     let response = client
         .patch(&url)
-        .header("Authorization", format!("Bearer {}", token.token))
-        .header("Content-Type", "application/json")
         .json(&request_body)
         .send()
         .context("Failed to send request")?;
```

Comment:
i know this wasn't changed in this PR but seems that context here is `Failed to update task stage`